Lib/test/test_py_compile.py

```diff
@@ -235,11 +235,12 @@ def pycompilecmd(self, *args, **kwargs):
         # assert_python_* helpers don't return proc object. We'll just use
         # subprocess.run() instead of spawn_python() and its friends to test
         # stdin support of the CLI.
+        opts = '-m' if __debug__ else '-Om'
```

Comment:
I'm not sure how this actually fixed things, but it seems to have done the trick 🤷‍♂️